pkg/service/rtcservice.go

```diff
@@ -288,7 +288,7 @@ func (s *RTCService) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		if _, ok := req.Message.(*livekit.SignalRequest_Ping); ok {
 			_ = sigConn.WriteResponse(&livekit.SignalResponse{
 				Message: &livekit.SignalResponse_Pong{
-					Pong: 1,
+					Pong: time.Now().UnixNano(),
```

Comment:
Not sure why this was set to 1. We were discussion using time. So, changed this to time.

Reply:
thanks for catching this. I'm not sure what clients are sending as the timestamp.. it might be milliseconds. This is not extremely clean in the protocol today. But we should probably standardize on something.